src/ResolvedName.rei

```diff
@@ -4,8 +4,17 @@ type eq = (t, t);
 
 let applyEquations: (~eqs: list(eq), t) => list(eq);
 
-let emitAllModuleItems: unit => string;
-let extendExportModules: t => unit;
+type moduleItemsEmitter;
+type moduleItems;
```

Comment:
i create those to types, `emitAllModuleItem`s is a `Hashtbt` and `extendExportModules` is an array of tuples (moduleName, exportModuleItem)